jobs/grafana/templates/bin/prometheus-dashboards

```diff
@@ -13,7 +13,7 @@ dashboard_files=$(ls <%= dashboard_file %>)
 for dashboard_file in ${dashboard_files[@]}; do
   filename=$(basename "${dashboard_file}")
   echo -e "Updating dashboard ${dashboard_file} at $(date)"
-  sed 's/\${<%= p('grafana.prometheus.datasource_input_name') %>}/<%= p('grafana.prometheus.datasource_name') %>/g' "${dashboard_file}" > "${DASHBOARDS_DIR}/${filename}"
+  sed 's/\${<%= p('grafana.prometheus.datasource_input_name') %>}/<%= p('grafana.prometheus.datasource_name') %>/g;s/\${<%= p('grafana.influxdb.datasource_input_name') %>}/<%= p('grafana.influxdb.datasource_name') %>/g' "${dashboard_file}" > "${DASHBOARDS_DIR}/${filename}"
```

Comment:
Can you please split this in 2 commands? One to replace the prometheus datasource and another one to replace the influxdb datasource. This will improve readability.